index.html

```diff
@@ -1,89 +1,257 @@
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <a href="#" class="btn">See project</a>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work_preview">
+                    <img class="project-img" src="./resources/images/Snapshoot Portfolio 2.png" alt="Project 1" />
+                </div>
+                <div class="work_detail">
+                    <h2 class="work_title">Multi-Post Stories</h2>
+                    <ul class="work_info">
+                        <li class="info_item">CANOPY</li>
+                        <li class="info_item">Back End Dev</li>
+                        <li class="info_item">2015</li>
+                    </ul>
+                    <p class="work_details_content">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="work_tech">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <a href="#" class="btn">See project</a>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work_preview">
+                    <img class="project-img" src="./resources/images/Snapshoot Portfolio 3.png" alt="Project 1" />
+                </div>
+                <div class="work_detail">
+                    <h2 class="work_title">Tonic</h2>
+                    <ul class="work_info">
+                        <li class="info_item">CANOPY</li>
+                        <li class="info_item">Back End Dev</li>
+                        <li class="info_item">2015</li>
+                    </ul>
+                    <p class="work_details_content">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="work_tech">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <a href="#" class="btn">See project</a>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work_preview">
+                    <img class="project-img" src="./resources/images/Snapshoot Portfolio 4.png" alt="Project 1" />
+                </div>
+                <div class="work_detail">
+                    <h2 class="work_title">Multi-Post Stories</h2>
+                    <ul class="work_info">
+                        <li class="info_item">CANOPY</li>
+                        <li class="info_item">Back End Dev</li>
+                        <li class="info_item">2015</li>
+                    </ul>
```

Comment:
- [ ]  As demanded by project requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly fix the display of this list in the 375px screen size to match design template. You can reduce the font size a bit so that the text won't break.
### Your design
![image](https://github.com/Afsheen96/Portfolio/assets/94551484/7f18d8b1-4733-454f-81b0-18765f451832)
### figma
![image](https://github.com/Afsheen96/Portfolio/assets/94551484/50b2b2f6-2874-468f-845d-8759db6da31e)